tests/cli/commands/test_info_command.py

```diff
@@ -129,6 +130,8 @@ def test_should_read_logging_configuration(self):
             assert "stackdriver" in text
 
     def tearDown(self) -> None:
+        for handler_ref in logging._handlerList[:]:
```

Comment:
In this test, we used the StackdriverTaskHandler which tries to connect to GCP in the `close()` method. To avoid this, I delete the handlers manually without calling this method. Similar to: https://github.com/apache/airflow/pull/13784/files#r561283952